frontend/uddhar-frontend/src/shared/components/InputField.jsx

```
@@ -0,0 +1,50 @@
+  placeholder,
+  disabled = false,
+}) => {
+  const isDisabled = name === "email" || disabled;
+  const grayedClass = isDisabled ? "bg-gray-300 text-black cursor-not-allowed" : "bg-white hover:border-gray-400 placeholder-gray-400";
+
+  return (
+    <div className="mb-6">
+      <div className="flex flex-col sm:flex-row sm:items-start gap-1 sm:gap-6">
+        <label className="block text-gray-700 text-sm font-medium min-w-[140px] pt-2">
+          {label}
+        </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)